calcom/cal.com
# feat: Remember the last tab while viewing a meeting

## What does this PR do?
This change adds a feature to the bookings page where if the user is viewing the details of a booking and presses the "Back to bookings" button on top, he would be re-directed to the last tab from where he came.

Fixes #9275

Loom Video Link: https://www.loom.com/share/52b97be0c63d4a5496b0a0c5b483a8e3

**Environment**: Staging(main branch)

## Type of change
- New feature (non-breaking change which adds functionality)

## How should this be tested?
- Go to the bookings tab.
- Select the unconfirmed tab
- Click on a booking
- Press the "Back to Meetings Tab"
- You should be redirected back to the "unconfirmed" tab or from whichever tab you were there initially.
